components/brave_wallet/browser/keyring_service.cc

```diff
@@ -1786,6 +1788,51 @@ void KeyringService::IsStrongPassword(const std::string& password,
   std::move(callback).Run(true);
 }
 
+void KeyringService::ValidatePassword(const std::string& password,
+                                      ValidatePasswordCallback callback) {
+  if (password.empty()) {
+    std::move(callback).Run(false);
+    return;
+  }
+
+  const std::string keyring_id = mojom::kDefaultKeyringId;
+
+  std::vector<uint8_t> salt(kSaltSize);
+  if (!GetPrefInBytesForKeyring(kPasswordEncryptorSalt, &salt, keyring_id)) {
+    std::move(callback).Run(false);
+    return;
+  }
+
+  // TODO(apaymyshev): move this call(and other ones in this file) to background
```

Comment:
Please help create an issue in brave/browser to track this, thanks.

Reply:
https://github.com/brave/brave-browser/issues/21803